mapie/conformity_scores/conformity_scores.py

```diff
@@ -63,38 +69,48 @@ def get_signed_conformity_scores(
 
         Parameters
         ----------
-        y: NDArray
-            Observed values.
+        X: ArrayLike
+            Observed feature values.
 
-        y_pred: NDArray
-            Predicted values.
+        y: ArrayLike
+            Observed target values.
+
+        y_pred: ArrayLike
+            Predicted target values.
 
         Returns
         -------
         NDArray
-            Unsigned conformity scores.
+            Signed conformity scores.
         """
 
     @abstractmethod
     def get_estimation_distribution(
         self,
+        X: ArrayLike,
         y_pred: ArrayLike,
-        conformity_scores: ArrayLike,
+        values: ArrayLike
     ) -> NDArray:
         """
         Placeholder for ``get_estimation_distribution``.
         Subclasses should implement this method!
 
         Compute samples of the estimation distribution from the predicted
-        values and the conformity scores.
+        targets and ``values``that can be either the conformity scores or
```

Comment:
TYPO: add a whitespace after `values`.

Reply:
SUGGESTION: it is note clear what `values` is with this description. It goes against common advice, but it would be better to make the description shorter.
I even suggest keeping the previous term: `conformity_scores`.

Reply:
`values` is not always the conformity scores : that is why I named it that way. Wouldn't it be more confusing to have a parameter `conformity_scores` that can be either the quantile of the conformity scores or the conformity scores (depending on where it is called) ?

Reply:
In both cases, there is a error in the docstring. It is changed with : `Compute samples of the estimation distribution from the predicted targets and ``values`` that can be either the conformity scores or the quantile of the conformity scores.`

Reply:
A quantile of conformity scores is still a conformity score.

Reply:
values" is too abstract in this situation.

Reply:
Done